style.css

```diff
@@ -128,37 +146,88 @@
   padding: 10px 12px;
   margin-right: 12px;
   border: 1px solid #8993a4;
-  font-family: "Inter";
+  font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 15px;
   line-height: 20px;
   color: #091e42;
 }
 
+form button {
+  margin: 15px;
+  background: #ff6b00;
+  color: #fff;
+  border: none;
+  width: 131px;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-bottom: 124px;
+  cursor: pointer;
+}
+
 .works button {
   border: none;
   width: 124px;
   background-color: #ff6b00;
-  font-family: "Inter";
+  font-family: "Inter", sans-serif;
   font-weight: 700;
   font-size: 17px;
   line-height: 24px;
-  color: #ffffff;
+  color: #fff;
   padding: 12px;
   margin-left: 24px;
   margin-top: 12px;
 }
 
+.art button {
+  border: none;
+  width: 327px;
+  text-align: center;
+  background-color: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  padding: 12px 12px;
+  margin-top: 22px;
+}
+
+.aboutMe button {
+  border: none;
+  width: 161px;
+  height: 46px;
+  text-align: center;
+  background-color: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  padding: 12px 12px;
+  margin-left: 24px;
+  margin-top: 22px;
+}
+
+form button:hover {
+  background-color: #fff;
+  color: #ff6b00;
+}
+
```

Comment:
- [ ] Please add `border` at hover interaction to this button to match the Figma design.
- [ ] Kindly add `active` button interaction because it,s a requirement and try as much as possible to stick to the Figma design.

Figma Design:
![image](https://user-images.githubusercontent.com/68796232/160876140-35aee595-34e4-4ac4-81bc-f2e22363347f.png)